pkg/api/query/v1.go

```diff
@@ -350,7 +350,7 @@ func (qapi *QueryAPI) queryRange(r *http.Request) (interface{}, []error, *api.Ap
 	}
 
 	step, apiErr := qapi.parseStep(r, qapi.defaultRangeQueryStep, int64(end.Sub(start)/time.Second))
-	if err != nil {
+	if apiErr != nil {
```

Comment:
Amazing. How did you catch this

Reply:
Laser eyes

![](https://media0.giphy.com/media/3o85xoi6nNqJQJ95Qc/giphy.gif)